src/setup-beam.js

```diff
@@ -423,7 +423,10 @@ async function get(url0, pageIdxs) {
         .get(
           url,
           {
-            headers: { 'user-agent': 'setup-beam' },
+            headers: {
+              'user-agent': 'setup-beam',
```

Comment:
Right, we need get_json function I suppose without the header (seemingly) for the content type a 401 will occur.

Reply:
Upon some investigation it's not the headers, but it consistently fails for windows, all the other envs are fine.

Reply:
Strange it worked at all. I skipped specifying the GITHUB_TOKEN env var in my gleam fork. So, this will _have_ to be optional. That is, we can only conditionally add the header otherwise we'll definitely break what's out there.

Reply:
> Right, we a need get_json function I suppose without the header (seemingly) for the content type a 401 will occur.

I implemented `jsonParse` which should output the non-JSON, the exception, and error out.